agent/lib/kontena/workers/volumes/volume_manager.rb

```diff
@@ -124,6 +125,10 @@ def volume_exist?(volume_name, driver)
       end
     end
 
+    def drivers_match?(expected, actual)
+      expected.split(':', 2)[0] == actual.split(':', 2)[0]
```

Comment:
Hmm.... if the `expected` from the Kontena volume driver actually contains a specific version, then perhaps this should reject volumes with an "incompatible" (?) version? Typically, I would expect the `expected` to just be a `plugin/name` without any version, and then the actual driver version can be ignored like this.